trajopt_optimizers/trajopt_sqp/test/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(GTest REQUIRED)
+
+macro(add_gtest test_name test_file)
+  add_executable(${test_name} ${test_file})
+  trajopt_target_compile_options(${test_name} PRIVATE)
+#  trajopt_clang_tidy(${test_name})
```

Comment:
Clang tidy is disabled.

Reply:
I had this off because it was failing, but it was just missing the NOLINT on the gtest macros. Its turned back on now for both packages.